packages/neutron-sdk/src/interchain_queries/queries.rs

```diff
@@ -40,7 +40,7 @@ pub fn query_kv_result<T: KVReconstruct>(
 }
 
 /// Queries interchain query result (raw KV storage values or transactions) from Interchain Queries Module
-fn get_interchain_query_result(
+pub fn get_interchain_query_result(
```

Comment:
why you decided to make it pub? if we want to expose it, we should make it more obvious how it's different from the `query_kv_result` above and highlight use cases for both funcs. I mean in this case we need to give it a better naming which will reflect that it returns a raw kv storage value for example and elaborate description

Reply:
Alright, I agree, I think I will add "raw" to the name of this function and cover it with some decent documentation.

Reply:
done